src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/RemoveProviderModal.jsx

```diff
@@ -6,6 +6,16 @@ export default function RemoveProviderModal({ onClose, provider, address }) {
   const title = 'Are you sure you want to remove this provider?';
   const content = (
     <>
+      <button
```

Comment:
I believe you can remove the `hideCloseButton` prop on the `Modal` component below to display a close icon, instead of needing to create one yourself.

Reply:
Done.